From 29c0b393283395c69ecdd747e960301e95c93bcf Mon Sep 17 00:00:00 2001
From: Felix Rath <felixm.rath@gmail.com>
Date: Sat, 15 May 2021 13:07:38 +0200
Subject: [PATCH] add missing <stdexcept> import

`std::logic_error` is used in this file, which resides in `<stdexcept>`, but was not imported before. This caused the build to fail, see, e.g., https://hydra.nixos.org/build/141997371/log.
---
 src/algo/crypt/lcg.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/algo/crypt/lcg.cc b/src/algo/crypt/lcg.cc
index 6c2a7945..66630a08 100644
--- a/src/algo/crypt/lcg.cc
+++ b/src/algo/crypt/lcg.cc
@@ -17,6 +17,7 @@
 
 #include "algo/crypt/lcg.h"
 #include <functional>
+#include <stdexcept>
 
 using namespace au;
 using namespace au::algo::crypt;
